.github/workflows/driver-schema-version.yml

```diff
@@ -1,36 +1,48 @@
-# NOTE: it is UNSAFE to run ANY kind of script when using the pull_request_target trigger!
-# DO NOT TOUCH THIS FILE UNLESS THE TRIGGER IS CHANGED.
-# See warning in https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
-
 name: Driver SCHEMA_VERSION checks
 on:
-  pull_request_target:
-    paths:
-      - 'driver/event_table.c'
+  pull_request:
+  push:
+    branches:
```

Comment:
I think we can retain the `paths` filter here instead of filtering using the `paths-filter` action; wdyt?

Reply:
Then we would need to distinguish what files have been changed (SCHEMA_VERSION or one of the others). Do we have a way to access that info? if so, I'm onboard.

Reply:
Mmh before, i was only triggering the check when `driver/event_table.c` was changed, and then i manually checked if SCHEMA_VERSION was touched.
I think we could do the same, ie: only trigger this ci check when 
```
- 'driver/event_table.c'
- 'driver/ppm_events_public.h'
- 'driver/event_stats.h'
- 'driver/syscall_*'
```
are changed, and then manually check if SCHEMA_VERSION was touched too.